.github/workflows/e2e-nvidia-l40s-x8.yml

```diff
@@ -22,6 +28,10 @@ jobs:
       label: ${{ steps.start-ec2-runner.outputs.label }}
       ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
     steps:
+      # We need to free disk space to avoid the "No space left on device" error on our GH runner
+      - name: Free disk space on Ubuntu runner
+        uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be # v1.3.1
```

Comment:
We actually have an in-house one of these: https://github.com/instructlab/instructlab/blob/main/.github/actions/free-disk-space/action.yml

Reply:
updated! was not aware. thanks!